maquetado/scss/partials/_styles.scss

```diff
@@ -2,9 +2,26 @@
 
 .container {
   font-family: $roboto-stack;
+  height: 100%;
+  overflow-y: auto;
   padding: 20px;
 }
 
+.main-wrapper {
+  display: flex;
+  flex-flow: column nowrap;
+  height: 100%;
+}
+
+.root {
```

Comment:
I'd recommend not applying styles to the `html` element. You can achieve what you want setting the styles to the `body` element and the main containers. 🙂